llap-tez/src/java/org/apache/hadoop/hive/llap/tezplugins/LlapTaskCommunicator.java

```diff
@@ -584,6 +581,17 @@ public void indicateError(Throwable t) {
         });
   }
 
+  /**
+   * This is a best effort action in case of an invalid LLAP_TOKEN. The communicator (LlapProtocolClientProxy) periodically
+   * refreshes tokens, but there is a relatively small time window (compared to the whole token maxLifetime),
+   * when the token is cancelled and it's not yet fetched from daemons. By the time we detect this problem, the current
+   * task attempt is already failed, but usually there are at least 3 task attempts before failing the task and dag, therefore
+   * fetching an LLAP_TOKEN synchronously here definitely solves the invalid token problem (as the next task attempt will use it).
+   */
+  private void handleInvalidToken(Throwable t) {
+    this.communicator.refreshToken();
```

Comment:
You don't need the argument ``Throwable t``

Reply:
ack, it's just a leftover